docs/common/craft-parts/explanation/overlays.rst

```diff
@@ -1,6 +1,8 @@
-****************
-``OVERLAY`` Step
-****************
+.. _overlays:
+
+************
+OVERLAY Step
```

Comment:
Does it need to be upper case? In https://documentation.ubuntu.com/rockcraft/en/stable/explanation/overlay-step/  we have it sentence case